apps/server/src/modules/account/services/account.service.ts

```diff
@@ -25,34 +26,51 @@ export class AccountService extends AbstractAccountService {
 	}
 
 	async findById(id: string): Promise<AccountDto> {
+		if (this.configService.get<boolean>('FEATURE_IDENTITY_MANAGEMENT_IS_PRIMARY') === true) {
```

Comment:
There was no additional commit on the `hpi-schul-cloud/commons` to fix the configuration parsing in the last four months. Because of this, I used the more save approach. So that even if we get a string with the content false the feature will not activate.

Reply:
This does not make it more save. 'true' === true evaluates to false.

Reply:
OK, that's true. That should not be fixed here, but I'll accept it.
The validation is fixed in config-module-options.js. The other repo never was the problem.